CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Trino
+
+## Contributor License Agreement ("CLA")
+
+In order to accept your pull request, we need you to [submit a CLA](https://github.com/trinodb/cla).
+
+## License
+
+By contributing to Trino, you agree that your contributions will be licensed under the [Apache License Version 2.0 (APLv2)](LICENSE).
+
+## Contribution process
+
+See the suggested [process for contributions](https://trino.io/development/process.html).
```

Comment:
trino and trino-python-client repositories put this file under `.github` repository. I would move to the same path.

Reply:
That seems like a bad idea .. a LOT less visible and against the common approach. I did see that on the other repos though .. 

Maybe move it but add mention in the README?

Reply:
Relates to https://github.com/trinodb/trino/pull/9168

Reply:
Cool .. thanks for pointing to that discussion.. will move

Reply:
Done